<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatreonDialog</class>
 <widget class="QDialog" name="PatreonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Become a Patron</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCommandLinkButton" name="gotoPatreon">
       <property name="text">
        <string>Visit https://www.patreon.com/bjorn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="alreadyPatron">
       <property name="text">
        <string>I'm already a patron!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="maybeLaterButton">
       <property name="text">
        <string>Maybe later</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatreonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatreonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maybeLaterButton</sender>
   <signal>clicked()</signal>
   <receiver>PatreonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
